src/Blorc.Core/Components/BlorcLayoutComponentBase.cs

```diff
@@ -138,12 +139,18 @@ protected override async Task OnAfterRenderAsync(bool firstRender)
                     }
                 }
             }
+
+            _hasRenderHandler = true;
+            while (_propertyChangedActionQueue.TryDequeue(out var propertyChangedAction))
+            {
+                propertyChangedAction();
+            }
         }
 
 
+
         protected override async Task OnInitializedAsync()
         {
-            await DocumentService.InjectBlorcCoreJsAsync();
```

Comment:
No longer needed?

Reply:
No, must / can do such a call in the Program startup configuring the webAssemblyHost

![image](https://user-images.githubusercontent.com/1785664/147353802-fb7ee8fd-34ef-4d1d-b6ea-38ad07fc576c.png)

or in the App just like this 

![image](https://user-images.githubusercontent.com/1785664/147353948-b648f24b-71df-41a3-9058-b4a0dab9043c.png)

+

![image](https://user-images.githubusercontent.com/1785664/147353969-cce0169f-878f-47bb-8179-6e28c033c33e.png)